float64 x
float64 y
float64 th
float64 v_x
float64 v_th
string MAC